english/end/end.md

```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (https://github.com/vladimirfomene/)
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.